src/Cli/dotnet/Commands/Run/RunCommand.cs

```diff
@@ -529,6 +529,13 @@ internal static void ThrowUnableToRunError(ProjectInstance project)
             projectFileOrDirectoryPath = Directory.GetCurrentDirectory();
         }
 
+        // Normalize path separators to handle Windows-style paths on non-Windows platforms
+        // First convert backslashes to forward slashes on non-Windows, then get the full path
+        if (Path.DirectorySeparatorChar != '\\')
```

Comment:
Consider adding a comment that we only support this for historical reasons to avoid a break. We probably don't strive to support `\` on Linux in all path arguments.

Reply:
In that case yeah let's not support that in dotnet test.
A comment would help.